config/config.go

```diff
@@ -593,6 +594,7 @@ var defaultConf = Config{
 		ServerMemoryQuota:     0,
 		MemoryUsageAlarmRatio: 0.8,
 		TCPKeepAlive:          true,
+		TCPNoDelay:            true,
```

Comment:
It needs more internal decisions on the default state of this switch. But if it defaults to false, I think we could merge this PR and you don't need to set it to false in the test.

Reply:
According to https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-networking.html, TCPNoDelay is positive as default.

Reply:
I saw that in your comment. I agreed that it is effective. But the decision may not be determined in a short time... I would suggest you merge the PR with an default-off-switch first: at least you will have an option that can be enabled.

Join the slack channel #sig-sql-infra for discussion with members.